R/lmap.R

```diff
@@ -63,12 +63,10 @@ lmap_at <- function(.x, .at, .f, ...) {
   lmap_helper(.x, sel, .f, ...)
 }
 
-lmap_helper <- function(.x, .ind, .f, ..., .else = NULL) {
-  if (is_formula(.f)) {
-    .f <- rlang::as_function(.f)
-  }
-  if (is_formula(.else)) {
-    .else <- rlang::as_function(.else)
+lmap_helper <- function(.x, .ind, .f, ..., .else = NULL, .error_call = caller_env()) {
+  .f <- rlang::as_function(.f, call = .error_call)
```

Comment:
Should also pass `arg`?

Reply:
It's always `.f`?

Reply:
I meant for resilience to refactoring / new usages, but that's fine as it is.